### PR TITLE
Added note in HACKING file 

### PR DESCRIPTION
**Pull request by rmescandon**
Note to supply a testing snap-seccomp file when detected security error while testing snapd

### CODE REVIEW

**Comment by mvo5** [comment]
I wonder if this should mention: `make hack`?

**Comment by mvo5** [comment]
But in any case :+1:

File: HACKING.md [hunk 1]
```diff
@@ -171,6 +171,11 @@ transfer it to the snappy system and then run:
 To debug interaction with the snap store, you can set `SNAP_DEBUG_HTTP`.
 It is a bitfield: dump requests: 1, dump responses: 2, dump bodies: 4.
 
+(note: In case you get some security profiles errors when trying to install or refresh a snap, 
+maybe you need to replace system installed snap-seccomp whith the one aligned to the snapd that
```
Comment (mvo5) [inline HACKING.md]: There is a typo here `whith` :)

**Comment by rmescandon** [comment]
Addressed comments. Thanks

**Pull request merged**